distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/PaxExamRuleTest.java

```diff
@@ -176,7 +174,7 @@ public void failingBeforeExam() {
             .stream()
             .anyMatch(failure -> failure.getMessage().contains(EXPECTED_BEFORE_EXAM_ERROR_MESSAGE));
 
-    assertThat(examFail && beforeExamFail);
+    assertTrue(examFail && beforeExamFail);
```

Comment:
This change goes against our unit testing guidelines.  https://codice.atlassian.net/wiki/display/DDF/Unit+Testing